pilot/pkg/networking/core/v1alpha3/listener_test.go

```diff
@@ -2549,35 +2549,35 @@ func verifyFilterChainMatch(t *testing.T, listener *listener.Listener) {
 				Name:           model.VirtualInboundCatchAllHTTPFilterChainName,
 				Type:           listenertest.MTLSHTTP,
 				HTTPFilters:    httpFilters,
-				NetworkFilters: []string{xdsfilters.MxFilterName, wellknown.HTTPConnectionManager},
+				NetworkFilters: []string{"istio_authn", xdsfilters.MxFilterName, wellknown.HTTPConnectionManager},
```

Comment:
very nit: could use `IstioNetworkAuthenticationFilter.Name` constant

Reply:
IMHO: it is far easier to read "istio_authn" ( and match against what I expect to see in the config dump ) compared with MxFilterName ( which needs another navigation to get the name ). But I agree, consistency first.